packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -93,14 +93,41 @@ const unthemedSeries = new Set<SeriesType>(['map-shape-background', 'map-line-ba
 export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
     private static readonly OPTIONS_CLONE_OPTS = new Set(['data', 'container']);
 
-    private static readonly FAST_PATH_OPTIONS = new Set<keyof AgChartOptions>(['data', 'width', 'height', 'container']);
-    private static isFastPathDelta(deltaOptions: DeepPartial<AgChartOptions>) {
+    private static readonly FAST_PATH_OPTIONS = new Map([
+        ['data', null],
+        ['width', null],
+        ['height', null],
+        ['container', null],
+        ['tooltip', new Map([['renderer', null]])],
+    ]);
+    private static isFastPathDelta(
+        deltaOptions: DeepPartial<AgChartOptions>,
+        options = ChartOptions.FAST_PATH_OPTIONS,
+        path = ''
+    ) {
+        const slowKeys = new Set<string>();
```

Comment:
This is only used for debugging, so we could initiate it lazily

```suggestion
        let slowKeys: Set<string> | undefined;
```